src/resolvelib/__init__.py

```diff
@@ -16,8 +16,8 @@
 
 from .providers import AbstractProvider
 from .reporters import BaseReporter
-from .resolvers import (
-    AbstractResolver,
+from .resolvers.abstract import AbstractResolver
+from .resolvers.criterion import (
```

Comment:
These should be imported from `.resolvers`, where all required members should be exported.